extensions/sql-bindings/src/test/common/util.test.ts

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as fs from 'fs';
+import * as should from 'should';
+import * as sinon from 'sinon';
+import * as constants from '../../common/constants';
+import { getErrorType, getUniqueFileName, TimeoutError, validateFunctionName } from '../../common/utils';
+
+describe('Utils', function (): void {
+	it('Should create unique file name', async () => {
+		let testFile = 'testFile';
+		let result = await getUniqueFileName(testFile);
+
+		should(result).be.equal(undefined);
```

Comment:
Why is the result undefined?

Reply:
The title of the test was not clear and I added description, this is due to no folderpath given.